components/ActionBar/ActionBar.js

```diff
@@ -0,0 +1,53 @@
+import { Container, MenuItem } from "./ActionBar.styled";
+import { LuSettings2 } from "react-icons/lu";
+import Image from "next/image";
+import { signIn, signOut } from "next-auth/react";
+
+export default function ActionBar({
+  dashboardIsOpen,
+  handleDashboardIsOpen,
+  session,
+}) {
+  function handleLoginButton() {
+    if (session) {
+      signOut();
+    } else {
+      signIn();
+    }
+  }
+
+  return (
+    <Container>
+      <MenuItem
+        onClick={() => {
+          handleLoginButton();
+        }}
+        $left
+        $isActive={true}
```

Comment:
In line 25 you work with the shorthand and in line 26 you don't.

I recommend being consistent.

```suggestion
        $left
        $isActive
```